### PR TITLE
craco 

### PR DESCRIPTION
**Pull request by MarijaRomanenkova**

### CODE REVIEW

File: craco.config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
```
Comment (viktarrudzenia) [inline craco.config.js:2]: extra spaces?

File: craco.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                    resources: [
+                      './src/scss-source/_mixins.scss',
+                      './src/scss-source/_variables.scss',
+                    ],
+                  },
+                },
+              ],
+            },
+          ],
+        },
```
Comment (viktarrudzenia) [inline craco.config.js:23]: is it worked well? is it okay to paste the necessary modules?

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^14.4.3",
     "axios": "^1.2.1",
     "classnames": "^2.3.2",
+    "craco-sass-resources-loader": "^1.1.0",
```
Comment (viktarrudzenia) [inline package.json:12]: maybe it will be better to install this dependency as devDependency?

**Review by viktarrudzenia — CHANGES_REQUESTED** [review]
looks good, just small comments

File: craco.config.js (hunk 2 above)
Comment (MarijaRomanenkova) [reply craco.config.js:23]: Yes, it works, I have romoved all imports of variables.scss and mixins and it works absolutely the same.

File: package.json (hunk 3 above)
Comment (MarijaRomanenkova) [reply package.json]: fixed. moved to devdependencies

File: craco.config.js (hunk 1 above)
Comment (MarijaRomanenkova) [reply craco.config.js]: fixed

File: package.json [hunk 4]
```diff
@@ -22,15 +22,15 @@
     "react-router-dom": "^6.4.5",
     "react-scripts": "5.0.1",
     "react-toastify": "^9.1.1",
-    "sass-resources-loader": "^2.2.5",
+    "sass-resources-loader": "^2.2.5",
```
Comment (viktarrudzenia) [inline package.json:25]: extra spaces ?

**Review by viktarrudzenia — APPROVED** [review]
looks good, could be merged into dev :)

**Pull request merged**